### ISSUE TITLE
Strategy to publish external modules for different versions of ITK

### ISSUE THREAD

**Issue by dyollb** [InsightSoftwareConsortium/ITKModuleTemplate#142]
Currently, it is tricky to know which version of an ITK external module can be used with a specific version of ITK.
Since often external modules are not updated on PyPi, users need to try until they find a version of ITK that (seems to) work.

It would be nice if there was a consistent strategy. Some ideas:

- in setup.py only the current minor versions are allowed, e.g. `itk >= 5.3.0, < 5.4.0`
- in setup.py only the current patch versions are allowed, e.g. `itk >= 5.2.1, < 5.2.2`

**Comment by SimonRit** [comment]
I agree with you, we should maybe fix the ITK version for a given module release. In my experience, some patch versions are incompatible, e.g., `5.3rc04.post3` and `5.3rc04.post4`.

**Comment by thewtex** [comment]
Yes, fixing to patch versions is conservative, but safest.
